sycl/doc/GetStartedGuide.md

```diff
@@ -439,7 +481,32 @@ clang++ -fsycl -fsycl-targets=nvptx64-nvidia-cuda-sycldevice \
   simple-sycl-app.cpp -o simple-sycl-app-cuda.exe
 ```
 
-This `simple-sycl-app.exe` application doesn't specify SYCL device for
+To build simple-sycl-app ahead of time for GPU, CPU or ACC devices, specify
+the target architecture:
+
+```-fsycl-targets=spir64_gen-unknown-unknown-sycldevice``` for GPU,  
+```-fsycl-targets=spir64_x86_64-unknown-unknown-sycldevice``` for CPU,  
+```-fsycl-targets=spir64_fpga-unknown-unknown-sycldevice``` for ACC.
+
+Multiple target architectures are supported.
+
+E.g., this command builds simple-sycl-app for GPU and CPU devices in
+ahead of time mode:
+
+```bash
+clang++ -fsycl -fsycl-targets=spir64_gen-unknown-unknown-sycldevice,spir64_x86_64-unknown-unknown-sycldevice simple-sycl-app.cpp -o simple-sycl-app-aot.exe
+```
+
+Additionally, user can pass specific options of AOT compiler to
+the DPC++ compiler using ```-Xsycl-target-backend``` option, see
+[Device code formats](CompilerAndRuntimeDesign.md#device-code-formats) for
+more. To find available options, execute:
+
+```bash
+clang++ -fsycl-help
+```
+
```

Comment:
```suggestion

To find available options of the GPU AOT compiler, run
`$ ocloc compile --help`

```

Reply:
Answered in the thread above.

Reply:
Likewise. I think `$ ocloc compile --help` should be mentioned in the docs.

Reply:
Updated.